rpm/quazip.spec

```diff
@@ -32,22 +32,20 @@ This packages contains the development files for building QuaZIP-based apps.
 %make_build
 
 %install
-rm -rf %{buildroot}
-make install/fast DESTDIR=%{buildroot}
-# Remove static library
-rm %{buildroot}%{_libdir}/libquazip5.a
+%make_install
 
 %post -p /sbin/ldconfig
 
 %postun -p /sbin/ldconfig
 
 %files devel
 %defattr(-,root,root,-)
-%{_includedir}/quazip5/*.h
-%{_libdir}/libquazip5.so
-%{_libdir}/pkgconfig/quazip.pc
-%{_libdir}/cmake/QuaZip5/QuaZip5Config.cmake
+%{_includedir}/QuaZip*/
+%{_libdir}/libquazip1-qt5.so
+%{_libdir}/pkgconfig/quazip1-qt5.pc
```

Comment:
The pkgconfig symbol could change here,  it could break some packages that use it.

Reply:
Yes. Already having a migration PR elsewhere.